### PR TITLE
GSoC 2024: add Threat Dragon ideas

### PR DESCRIPTION
**Pull request by jgadsden**
I hope it is not too late to add Threat Dragon ideas to the GSoC 2024

These ideas are all at the medium level, and are to do with the threat engine features that have yet to be carried over from the rewrite of Threat Dragon

### CODE REVIEW

**Comment by jgadsden** [comment]
@kingthorin I hope it is not too late to add Threat Dragon to the GSoC 2024

File: pages/initiatives/gsoc/gsoc2024ideas.md [hunk 1]
```diff
@@ -660,3 +660,41 @@ Repositories:
 
 * [Sam Stepanyan](mailto:sam.stepanyan@owasp.org)
 * [Ali Razmjoo](mailto:ali.razmjoo@owasp.org)
+
+### [OWASP Threat Dragon](https://owasp.org/www-project-threat-dragon/)
```
Comment (kingthorin) [inline pages/initiatives/gsoc/gsoc2024ideas.md:664]: Add Threat Dragon to the jump list at line 10 as well

File: pages/initiatives/gsoc/gsoc2024ideas.md [hunk 2]
```diff
@@ -660,3 +660,41 @@ Repositories:
...
+
+OWASP Threat Dragon is a modeling tool used to create threat model diagrams as part of a secure development lifecycle.
+
+![Difficulty: Medium](https://img.shields.io/badge/difficulty-medium-orange)
+![Preferred for "Medium" GSoC 2032 project](https://img.shields.io/badge/medium%20size%20(~175h)-preferred-green)
```
Comment (kingthorin) [inline pages/initiatives/gsoc/gsoc2024ideas.md:669]: Time travel?

**Comment by jgadsden** [comment]
thanks @kingthorin , and I have fixed up the year and added Threat Dragon to the jump list

**Review by kingthorin — APPROVED** [review]

**Comment by kingthorin** [comment]
Live: https://owasp.org/www-community/initiatives/gsoc/gsoc2024ideas#owasp-threat-dragon

**Pull request merged**